### PR TITLE
boveri Diagnosis node updates

### PR DESCRIPTION
**Pull request by ganjigajanan**

### CODE REVIEW

File: gdcdictionary/schemas/diagnosis.yaml [hunk 1]
```diff
@@ -731,6 +750,8 @@ properties:
       - Stage IIC
       - Stage III
       - Stage IIIA
+      - Stage IIIA1
```
Comment (wteouchicago) [inline gdcdictionary/schemas/diagnosis.yaml:753]: DICT93 only lists Stage IIIA2 as the new enum. This doesn't _seem_ wrong though - just verifying.

Comment (ganjigajanan) [reply gdcdictionary/schemas/diagnosis.yaml:753]: @tlicht3 added that along with StageIIIA2

**Comment by wteouchicago** [comment]
looks good to me.

**Review by wwysoc2 — APPROVED** [review]
lgtm

**Review by wteouchicago — APPROVED** [review]

**Pull request merged**